### PR TITLE
plugins updated

### PR DESCRIPTION
**Pull request by kirannonstop**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -10,17 +10,18 @@ dependencies:
...
+  fake_cloud_firestore: ^2.4.1+1
+  firebase_analytics: ^10.4.5
+  firebase_auth: ^4.9.0
+  firebase_core: ^2.15.1
+  firebase_crashlytics: ^3.3.5
+  firebase_dynamic_links: ^5.3.5
+  firebase_messaging: ^14.6.7
+  firebase_remote_config: ^4.2.5
+  firebase_storage: ^11.2.6
+  cloud_firestore_platform_interface: ^5.16.0
```
Comment (ProjectAJ14) [inline pubspec.yaml]: Why do need a `UNLISTED` package i.e `cloud_firestore_platform_interface` ?

**Review by ProjectAJ14 — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (kirannonstop) [reply pubspec.yaml]: I apologies for it, As per [this](https://github.com/firebase/flutterfire/issues/10140) and [this](https://stackoverflow.com/questions/74861470/error-method-not-found-fallthrougherror-throw-fallthrougherror) issues I thought we also need to add it, yes we don't require this package here.

Comment (ProjectAJ14) [reply pubspec.yaml]: Please remove it

Comment (kirannonstop) [reply pubspec.yaml]: Removed.

**Review by ProjectAJ14 — APPROVED** [review]

**Pull request merged**